draft-ietf-rtcweb-jsep.xml

```diff
@@ -1859,6 +1859,20 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             This option allows the endpoint to significantly reduce the amount of
             audio bandwidth it receives, at the cost of some fidelity, depending on
             the quality of the remote VAD algorithm.</t>
+            <t>If the "VoiceActivityDetection" option is specified, with a value
+            of "false", the browser MUST NOT omit "CN" codecs.
+            For codecs that have their own internal silence suppression support,
+            the appropriate fmtp parameters for that codec MUST be
+            specified to indicate that silence suppression for received audio is
+            not desired. For example, when using the Opus codec, the
+            "usedtx=0" parameter would be specified in the offer.</t>
+            <t>Note that setting the "VoiceActivityDetection" parameter on an
```

Comment:
suggest "... parameter when generating an offer is a request to receive audio with silence suppression. It has no impact on whether the local endpoint does silence suppression for the audio it sends."